src/test/clojure/xtdb/pgwire_test.clj

```diff
@@ -1982,8 +1985,36 @@ ORDER BY t.oid DESC LIMIT 1"
 
         (t/is (= [{"_id" 2, "f" 2.0, "i" 2} {"_id" 1, "f" 1.0, "i" 1}] (rs->maps rs)))))))
 
+(deftest test-primitive-array
+  (t/testing "primitive array"
```

Comment:
nit: don't need the `t/testing` here - you'll get the same info from the test name